### PR TITLE
Playground for testing Review feature

### PR DESCRIPTION
**Pull request by DrewML**
Just wanted a place we can see the different states for the review feature. Can someone approve this, and someone else request changes?

@sindresorhus @hkdobrev @SpaceK33z @paulmolluzzo

### CODE REVIEW

File: extension/content.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /* globals gitHubInjection, pageDetect, diffFileHeader, addReactionParticipants, addFileCopyButton, enableCopyOnY, addBlameParentLinks, showRealNames, markUnread */
 
+// wookie noise
```
Comment (hkdobrev) [inline extension/content.js]: Could you use caps here?

File: extension/content.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+
```
Comment (hkdobrev) [inline extension/content.js]: Could you 🔥  this line?

File: extension/content.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
 /* globals gitHubInjection, pageDetect, diffFileHeader, addReactionParticipants, addFileCopyButton, enableCopyOnY, addBlameParentLinks, showRealNames, markUnread */
```
Comment (hkdobrev) [inline extension/content.js]: I don't think that's needed anymore.

File: extension/content.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
...
 
+// wookie noise
+
 'use strict';
 const {ownerName, repoName} = pageDetect.getOwnerAndRepo();
 const repoUrl = `${ownerName}/${repoName}`;
```
Comment (sindresorhus) [inline extension/content.js]: Needs moar unicorn!

File: extension/content.js (hunk 1 above)
Comment (sindresorhus) [inline extension/content.js]: I think we should go with:

> wOoKiE nOisE

**Pull request closed without merging**